pwndbg/commands/ipython_interactive.py

```diff
@@ -46,7 +46,208 @@ def ipi() -> None:
+    '''
+    try:
+        addr_int = int(addr, 16) if isinstance(addr, str) else int(addr)
+    except ValueError:
+        print(f"Invalid address: {addr!r}. Use hex string or integer.")
+        return None
+    return write(addr_int, data)
+
+
+def h(addr, length):
+    '''
+    Alias for pwndbg's `hexdump` command.
+    Displays memory contents in a formatted hexdump view.
+
+    This command reads memory at a specified address (or module name)
+    and prints a traditional hexdump with configurable width, grouping,
+    and endianness options. It respects global pwndbg configuration
+    parameters such as line width, group width, and size limits.
+
+    Args:
+        address (int or str): Starting memory address to dump, or a module name.
+                              If a module name is given, the first mapped page
+                              for that module is used.
+                              Defaults to the current stack pointer (`$sp`).
+        count (int): Number of bytes to dump. Defaults to the value of
+                     `hexdump-bytes` (configurable). May also be interpreted
+                     as an end address if larger than the given `address`.
+
+    Returns:
+        None. Prints hexdump lines directly.
+
+    Behavior:
+        - If `count` exceeds the configured `hexdump-limit-mb`, an error is raised.
+        - If `address` exceeds the architecture's maximum pointer value,
+          it is truncated to fit.
+        - If the same command is repeated (`hexdump.repeat` is set),
+          it resumes dumping from the last stored address.
+        - Group width defaults to the architecture’s pointer size when set to -1.
+        - Endianness within groups is controlled by `hexdump-group-use-big-endian`.
+
+    Raises:
+        ValueError: If the requested `count` exceeds the configured maximum limit.
+        Prints warnings or errors if the memory cannot be accessed.
+
+    Notes:
+        - Results are paged based on pwndbg\'s config options:
+          * `hexdump-width`: number of bytes shown per line.
+          * `hexdump-group-width`: number of bytes per grouping.
+          * `hexdump-group-use-big-endian`: flips group display endianness.
+          * `hexdump-limit-mb`: maximum allowed dump size.
+        - `hexdump.last_address` and `hexdump.offset` are updated internally
+          to support consecutive calls.
+    '''
+    try:
+        addr_int = int(addr, 16) if isinstance(addr, str) else int(addr)
+    except ValueError:
+        print(f"Invalid address: {addr!r}. Use hex string or integer.")
+        return None
+    hexdump(addr_int, length)
+
+
+@wraps(vmmap.get)
+def vv():
+    '''
+    Wrapper alias for pwndbg.vmmap.get.
+
+    Returns the virtual memory map for the current process.
+
+    Returns:
+        Typically a list of memory mapping entries as provided by pwndbg.vmmap.get.
+
+    Raises:
+        None.
+    '''
+    return vmmap.get()
```

Comment:
This is awful. You know this could be `from pwndbg.aglib.vmmap import get as vv`? lol...